gfx_window.h

```diff
@@ -58,6 +59,9 @@ bool gfxWindowIsKeyReleased(GfxWindow window, uint32_t key_code);
 bool gfxWindowIsCloseRequested(GfxWindow window);
 bool gfxWindowIsMinimized(GfxWindow window);
 bool gfxWindowIsMaximized(GfxWindow window);
+bool gfxRegisterDropCallback(
+    GfxWindow window, void (*callback)(char const*, uint32_t, void *), void *data = nullptr);
+bool gfxUnregisterDropCallback(GfxWindow window);
```

Comment:
To handle the drag and drop event a callback into user code is needed. I added 2 new functions to register and then de-register these callbacks

Reply:
Should be `gfxWindowRegisterDropCallback()` and `gfxWindowUnregisterDropCallback()`.

Reply:
Yes, I dont know why I hadnt seen that, corrected now